### ISSUE TITLE
Triple number of Validation Set is zero?

### ISSUE THREAD

**Issue by chuanyang-Zheng** [dice-group/Convolutional-Complex-Knowledge-Graph-Embeddings#4]
When I run the search.py, it says that the validation set triples is zero. Do you combine the validation set and training set?
![2](https://user-images.githubusercontent.com/56055931/103471551-b0c7b300-4dbc-11eb-8c66-ada4cb6b91db.PNG)

**Comment by Demirrr** [comment]
Hello @chuanyang-Zheng,

we retrained the model on the combination of the training and validation sets, after the best hyperparameter configuration  were determined on the training set. As you can see in [search.py](https://github.com/dice-group/Convolutional-Complex-Knowledge-Graph-Embeddings/blob/0f25815988b5912a984b204cca5216ebef996693/search.py#L18), `train_plus_valid` is set `True` by default. If you wish to use the validation set, please set this parapameter to `False`.
